check.go

```diff
@@ -125,10 +133,18 @@ func RepairEverything() *RepairOptions {
 // Check returns a list of problems with what's in the store, as a whole.  It can be very expensive
 // to call.
 func (s *store) Check(options *CheckOptions) (CheckReport, error) {
-	var ignoreChownErrors bool
+	var ignore checkIgnore
 	for _, o := range s.graphOptions {
 		if strings.Contains(o, "ignore_chown_errors") {
-			ignoreChownErrors = true
+			ignore.ownership = true
+		}
+		if strings.Contains(o, "force_mask") {
```

Comment:
(Overlay also sets `force_mask` if `isNetworkFileSystem`. At some point it may be easiest to let the driver determine the right `checkIgnore` value.

Automatically-setting `force_mask` if `isNetworkFileSystem` only happens with a warning, so not handling that here seems fair enough.)